docs/Installation.rst

```diff
@@ -7,89 +7,78 @@
-`UFS-SRW wiki <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_
-and include platform name and compiler version.
+Users working on unsupported platforms will need to install the HPC-Stack on their system and can do so following
+the instructions in the `HPC-Stack User's Guide <https://hpc-stack.readthedocs.io/en/latest/>`_.
 
 ============================
 Obtaining and Installing UPP
 ============================
 
-Building and running UPP V9.0.0 has been tested on the following platforms using pre-configured libraries.
+Building and running UPP V10.0.12 has been tested and is supported on the following pre-configured platforms.
 
 +---------------+----------------------+
 | System        | Compiler and Version |
 +===============+======================+
-| NCAR Cheyenne | Intel 19.1.1         |
+| NCAR Cheyenne | Intel 2021.2         |
 |               +----------------------+
-|               | GNU 9.1.0            |
 |               | GNU 10.1.0           |
 +---------------+----------------------+
 | NOAA Hera     | Intel 18.0.5.274     |
 +---------------+----------------------+
+| NOAA Orion    | Intel 2018.4         |
++---------------+----------------------+
 
-Move to the directory where you want to clone and build UPP and clone the repository into the directory
-UPP.
+Move to the directory where you want to install UPP and clone the repository.
 
 .. code-block:: console
 
     git clone -b branch-or-tag-name https://github.com/NOAA-EMC/UPP
 
 where, ``branch-or-tag-name`` is the release branch or tag you wish to clone.
 
-Move into the top level UPP directory and create and move into the build directory. Then build the UPP code
-using the cmake utility.
-The path ``INSTALL_PREFIX`` should point to the location of the pre-installed NCEP libraries.
+Move to the directory with the build script and build the UPP.
 
 .. code-block:: console
 
-    cd UPP
-    mkdir build && cd build
+    cd UPP/tests
 
-    cmake .. -DCMAKE_INSTALL_PREFIX=.. -DCMAKE_PREFIX_PATH=${INSTALL_PREFIX}
-    make install
+    ./compile_upp.sh
 
 .. note::
-   To build in debug mode, you can add :bolditalic:`-DCMAKE_BUILD_TYPE=Debug` to the cmake command.
+   To build in debug mode, you can add :bolditalic:`-DCMAKE_BUILD_TYPE=Debug` to the *cmake_opts*
+   parameter in the :bolditalic:`compile_upp.sh` script.
    This removes compiler optimization flags and adds -g to the fortran compilation. You can also use
    :bolditalic:`-DCMAKE_BUILD_TYPE=RELWITHDEBINFO`, which gives the -g, but keeps the -O2 optimization
    for the fortran compilation.
 
 Move back to the top level UPP directory and create a directory for the CRTM fix files to be unpacked
 in. Download the fix files from the Github `release page
-<https://github.com/NOAA-EMC/UPP/releases/tag/upp_v9.0.0>`_ or use the wget command. Unpack the
+<https://github.com/NOAA-EMC/UPP/releases/tag/upp_v10.0.12>`_ or use the wget command. Unpack the
```

Comment:
@fossell This tag exists as released on March 12 2022. Should we be using 10.0.13 instead?